core/store/src/trie/shard_tries.rs

```diff
@@ -41,6 +41,8 @@ struct ShardTriesInner {
     state_snapshot: Arc<RwLock<Option<StateSnapshot>>>,
     /// Configures how to make state snapshots.
     state_snapshot_config: StateSnapshotConfig,
+    /// Tells which ShardUId prefix should be used instead when accessing db State column.
```

Comment:
nit: It would be nice to give full nice explanation of this logic in comment on `map_shard_uid` and link to it from all other relevant pieces. something along the lines of the following should do the trick:
```// Please see the comment on  [`map_shard_uid`] ```